core/amber/src/main/scala/edu/uci/ics/texera/web/resource/aiassistant/AiAssistantResource.scala

```diff
@@ -86,4 +100,44 @@ class AIAssistantResource {
        |- For the second situation: you return strictly according to the format " -> type", without adding any extra characters. No need for an explanation, just the result -> type is enough!
      """.stripMargin
   }
+
+  @POST
+  @RolesAllowed(Array("REGULAR", "ADMIN"))
+  @Path("/getArgument")
```

Comment:
`getArgument` does not reflect this endpoint's behavior. use `/annotate-argument`?